Corrected /etc/nvidia/gridd.conf

ppavlik opened the pull request:
The NVIDIA driver actually reads /etc/nvidia/gridd.conf and not /etc/nvidia/gridd.conf.template

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Commit:
Corrected /etc/nvidia/gridd.conf

The NVIDIA driver actually reads /etc/nvidia/gridd.conf and not /etc/nvidia/gridd.conf.template

Commit:
Updated based on info provided in "man nvidia-gridd"
